### DISCUSSION TITLE
No branch for Kirkstone?

### DISCUSSION THREAD

**Discussion by zeenix** [meta-rust/meta-rust#428] [Q&A]
I'm new to yocto so this is probably a silly question but why is there no branch for Kirkstone? :thinking: 

Given this:

> When poky creates releases, we will create a branch with the same name as the poky release

could it be that it was just forgotten?

**Comment by srwalter** [comment]
Typically we only create a branch once the "master" branch no longer works, i.e. we have to make an incompatible change.  kirkstone is still supported from the master branch, so creating a separate kirkstone branch would just be extra work with no benefit
